File: Web-app-oppgave-2/Models/Maaltid.cs
using System.ComponentModel.DataAnnotations;

namespace Web_app_oppgave_2.Models
{
    public class Maaltid
    {
        [Key]
        public int MaaltidId { get; set; }
        public string Navn { get; set; }
        public string Beskrivelse { get; set; }
        //public byte[] Bilde { get; set; }
        public double Pris { get; set; }
    }
}

File: Web-app-oppgave-2/ClientApp/src/models/maaltid.ts
export class Maaltid{
  maaltidId?:number;
  navn:string;
  beskrivelse:string;
  bilde?:any;
  pris:number;

  constructor(navn:string, beskrivelse:string,pris:number){
    this.navn = navn;
    this.beskrivelse = beskrivelse;
    //this.bilde = bilde;
    this.pris = pris;
  }

}
